frontend/src/components/AppLayout.tsx

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
@acouch wonder if it's a little cleaner to add these under `/components/app` folder rather than add it to the filename but maybe there's a Next'ism here

Build up all our search components in `/components/app` for separation - then when we've converted the whole application to app router, move all the `components/app` files into `components` folder

Reply:
In the tech spec Sawyer mentioned just use shared components for both the page router layout and the app router layout.

I think that would mean we should just reuse the existing Footer, Header, and Grants component - and not create separate AppFooter/AppHeader etc...